exercises/ocr-numbers/OcrNumbersTest.swift

```diff
@@ -0,0 +1,187 @@
+import XCTest
+
+// Apple Swift version 2.1
+
+class OcrNumbersTest: XCTestCase {
+    
+    func testRecognizeZero() {
+        let text =  " _ \n" +
+                    "| |\n" +
+                    "|_|\n" +
+                    "   "
+        XCTAssertEqual("0", try? OCR(text).convert())
```

Comment:
Let's call this 'converted' if is not mutating to start following the new naming guidelines. (Well need to revise the rest of the problems once that comes out. )